fdbserver/RESTKmsConnector.actor.cpp

```diff
@@ -1142,7 +1142,7 @@ void testGetEncryptKeysByDomainIdsRequestBody(Reference<RESTKmsConnectorCtx> ctx
 	for (int i = 1; i < nKeys; i++) {
 		EncryptCipherDomainId domainId = getRandomDomainId();
 		EncryptCipherDomainNameRef domainName = domainId < 0
-		                                            ? StringRef(arena, std::string(FDB_DEFAULT_ENCRYPT_DOMAIN_NAME))
+		                                            ? StringRef(arena, FDB_DEFAULT_ENCRYPT_DOMAIN_NAME_REF.toString())
```

Comment:
You shouldn't convert this to a string, `StringRef` has a constructor that can copy directly from another `StringRef` type.